src/geekbrains/lesson3Practical/Main.java

```diff
@@ -1,9 +1,172 @@
 package geekbrains.lesson3Practical;
 
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+
 public class Main {
 
+    static Random rndGenerator = new Random();
+    static Scanner scanUser = new Scanner(System.in);
+
     public static void main(String[] args) {
 
+        Task1();
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.